docs/Reference/ID-Prefix-Reservation.md

```diff
@@ -0,0 +1,156 @@
+---
+# required metadata
+
+title: ID Prefix Reservation Reference | Microsoft Docs
+author: dajaco
+ms.author: dajaco
+manager: unnir
+ms.date: 10/9/2017
+ms.topic: reference
+ms.prod: nuget
+#ms.service:
+ms.technology: null
+#ms.assetid: 
+
+# optional metadata
+
+description: Package ID Prefix Reservation feature description and author guide.
+keywords: NuGet package ID, prefix, reservation
+#ROBOTS:
+#audience:
+#ms.devlang:
+ms.reviewer:
+- anangaur
```

Comment:
use github userid for all of us, for example, use karann-msft for me

Reply:
Updated